src/status_im/contexts/shell/share/profile/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im.contexts.shell.share.profile.view
```

Comment:
Created a new file share/profile/view and moved the profile code below that was previously in share/view to improve code organization and readability.